<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1160</y>
      <width>1600</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(76,76,76);</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="TimerSetter" name="lbDateTime">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>2021-10-29  18:00:05</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCapture">
       <property name="text">
        <string>截图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQJ1">
       <property name="text">
        <string>全景1</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQJ2">
       <property name="text">
        <string>全景2</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnZoomRestore">
       <property name="text">
        <string>放大</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnHealthLed">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/resource/green.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnHealthState">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>华文黑体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255,255,255);
border:0px;
</string>
       </property>
       <property name="text">
        <string>健 康 状 态</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnMainWgt">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>华文黑体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    color:rgb(0,0,0);
     border-image: url(:/resource/button1.png);
} </string>
       </property>
       <property name="text">
        <string>首   页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnQuanjing">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>华文黑体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    color:rgb(0,0,0);
     border-image: url(:/resource/button1.png);
}  </string>
       </property>
       <property name="text">
        <string>全   景</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnZhouShi">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>华文黑体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    color:rgb(0,0,0);
     border-image: url(:/resource/button1.png);
} </string>
       </property>
       <property name="text">
        <string>周    视</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnNVR">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>华文黑体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    color:rgb(0,0,0);
     border-image: url(:/resource/button1.png);
} </string>
       </property>
       <property name="text">
        <string>NVR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnExit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>华文黑体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    color:rgb(0,0,0);
     border-image: url(:/resource/button1.png);
} </string>
       </property>
       <property name="text">
        <string>退    出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1145</y>
      <width>1600</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(95, 95, 95);</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimerSetter</class>
   <extends>QLabel</extends>
   <header location="global">timersetter.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>BtnMainWgt</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnClickedBtnMainWgt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1340</x>
     <y>1170</y>
    </hint>
    <hint type="destinationlabel">
     <x>1315</x>
     <y>1128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnZhouShi</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnClickedBtnZhouShi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1443</x>
     <y>1168</y>
    </hint>
    <hint type="destinationlabel">
     <x>1408</x>
     <y>1125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnClickedBtnExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1564</x>
     <y>1175</y>
    </hint>
    <hint type="destinationlabel">
     <x>1512</x>
     <y>1123</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnClickedBtnMainWgt()</slot>
  <slot>OnClickedBtnExit()</slot>
  <slot>OnClickedBtnZhouShi()</slot>
 </slots>
</ui>
